src/coreclr/jit/compiler.cpp

```diff
@@ -7451,8 +7452,8 @@ void Compiler::compJitStats()
 
 void Compiler::compCallArgStats()
 {
-    GenTree* args;
-    GenTree* argx;
+    GenTree* args = nullptr;
```

Comment:
where is 'args' and 'argx' used in this method? Looks like they can be just deleted.

Reply:
Thank you for reviewing! I've fixed this all.